src/modules/iasql_functions/rpcs/iasql_create_review.ts

```diff
@@ -0,0 +1,97 @@
+import { IasqlFunctions } from '..';
+import { RpcInput } from '../..';
+import {
+  Context,
+  PostTransactionCheck,
+  PreTransactionCheck,
+  RpcBase,
+  RpcResponseObject,
+} from '../../interfaces';
+
+/**
+ * Method that generates creates the markdown to be shared or used in any version control system
+ */
+export class IasqlCreateReview extends RpcBase {
+  /** @internal */
+  module: IasqlFunctions;
+  /** @internal */
+  preTransactionCheck = PreTransactionCheck.NO_CHECK;
+  /** @internal */
+  postTransactionCheck = PostTransactionCheck.NO_CHECK;
+  /** @internal */
+  inputTable: RpcInput = {
+    title: { argType: 'varchar', default: 'New Review' },
+    description: { argType: 'varchar', default: 'Review description' },
+  };
+  /** @internal */
+  outputTable = {
+    markdown: 'text',
+  } as const;
+
+  documentation = {
+    description: 'Generate review markdown for latest transaction',
+    sampleUsage: "SELECT * FROM iasql_create_review('My review title', 'Meaningful description')",
+  };
+
+  /** @internal */
+  call = async (
+    _dbId: string,
+    _dbUser: string,
+    ctx: Context,
+    title: string,
+    description: string,
+  ): Promise<RpcResponseObject<typeof this.outputTable>[]> => {
+    let preview: any[];
+    let transactionSql: { sql: string }[];
+    let markdown: string;
+    try {
+      preview = (await ctx.orm.query('SELECT * FROM iasql_preview();')) ?? [];
```

Comment:
Why not call the RPCs directly here?

Reply:
this way we will make sure that the pre and post-transaction hooks are called properly